memtier_benchmark.cpp

```diff
@@ -694,15 +694,15 @@ static int config_parse_args(int argc, char *argv[], struct benchmark_config *cf
                     break;
                 case o_key_stddev:
                     endptr = NULL;
-                    cfg->key_stddev = (unsigned int) strtof(optarg, &endptr);
+                    cfg->key_stddev = strtoull(optarg, &endptr, 10);
```

Comment:
I don't like the loss of fractional part. Specifically in stddev.
But anyway I think the problem is just the wrong use of 32bit float instead of double (although even 32bit float can hold values much larger than 32bit integer).
I.e. `strtod` should solve it and keep both thr fractional part and a huge range.

Reply:
i missed the fact that we had casting (`(unsigned int)`), so we didn't have fractional part anyway.
but that casting is actually the problem (not the `strtof`), so it was maybe enough to just drop the casting.
but if we change these lines, let's got with `strtod` (changing to `strtoull` would be a breaking change, since old commands that used to work, will now error)